portfolio/src/main/webapp/script.js

```diff
@@ -75,6 +80,27 @@ function openSidebar(el, button){
     button.classList.add('closeButton');
 }
 
+function toggleBottomBar() {
+    let buttonDiv = document.getElementById('authentication');
+    let button = document.getElementById('votingToggle');
+    let el = document.getElementById("chartpart");
+    if (isBottombarOpen === false){
+        if (isSidebarOpen){
+            toggleSidebar();
+            isSidebarOpen = !isSidebarOpen;
+        }
+        el.style.height = "260px";
+        button.innerText = "Close";
+        buttonDiv.classList.remove('fixToBottom');
+        buttonDiv.classList.add('moveUp');
+    } else {
+        el.style.height = "0";
+        button.innerText = "Show voting!";
+        buttonDiv.classList.add('fixToBottom');
+        buttonDiv.classList.remove('moveUp');
+    }
+    isBottombarOpen = !isBottombarOpen;
```

Comment:
In general, it's more readable to modify your state variables first, then run their side effects afterwards. This helps folks reading your code sequentially know what's going to come next and improves readability. So toggling isBottombarOpen should probably be the first line of this function. You'll have to invert the if statements.